frontend/css/playlists.less

```diff
@@ -1,11 +1,60 @@
 @playlistBackgroundColor: #fafafa;
 @descriptionLines: 3;
 
+#playlists-page .playlist-view-options {
+  display: flex;
+  margin-top: 0.8rem;
+  gap: 1rem;
+
+  .playlist-sort-controls {
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+  }
+}
+
 #playlists-container {
   display: flex;
   flex-wrap: wrap;
   padding-top: 1em;
 
+  &.list-view {
+    flex-direction: column;
+  }
+
+  /* Pure CSS multiline ellipsis by Sagi Shrieber:
+			   http://hackingui.com/a-pure-css-solution-for-multiline-text-truncation/
+			*/
+  .description {
+    overflow: hidden;
+    position: relative;
+    line-height: 1.2em;
+    max-height: @descriptionLines * 1.2em;
+    text-align: justify;
+    padding-right: 2em;
+    display: -webkit-box;
+    line-clamp: @descriptionLines;
+    -webkit-line-clamp: @descriptionLines;
+    -webkit-box-orient: vertical;
+
+    &:before {
+      content: "...";
+      position: absolute;
+      right: 1em;
+      bottom: 0;
+      display: none;
+    }
+    &:after {
+      content: "";
+      position: absolute;
+      right: 0;
+      width: 1em;
+      height: 1em;
+      margin-top: 0.2em;
+      background-color: inherit;
+    }
```

Comment:
I'm not certain we need these. I believe it is made as a fallback fro the line-clamp property if that's not implemented. I don't think it combines with the line-clamp stuff, and it shows two sets of ellipsis once set up properly, so we probably just want to drop it.
 
If we still want to use them, there are currently some issues with these pseudo elements:
- display:none on :before prevents rendering of the element
- :after is missing `bottom:0` for placement (reading the code comments on the article) and the `right` position needs to be the same as :before
- :after background-color cannot be inherit, which results in transparent. It should be the same color as the card background (`white` in this case)